apps-rendering/src/server/page.tsx

```diff
@@ -68,7 +68,7 @@ const styles = (format: ArticleFormat): string => `
         margin: 0;
         font-family: 'Guardian Text Egyptian Web';
         overflow-x: hidden;
-        line-height: 1.5;
+        line-height: 1.4;
```

Comment:
According to the design system, Guardian Text Egyptian Web should be line height 1.3: https://github.com/guardian/design-system-advocates/issues/1

Reply:
According to [WCAG, it should be 1.5](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html)

Reply:
Based on previous conversations we _think_ we're ok to proceed here. @mxdvl has added comments to the code to help explain here: https://github.com/guardian/csnx/pull/764#event-9974242107